rust/datafusion/src/physical_plan/functions.rs

```diff
@@ -118,6 +118,14 @@ pub enum BuiltinScalarFunction {
     Length,
     /// concat
     Concat,
+    /// character_length
+    CharacterLength,
+    /// lower
+    Lower,
+    /// upper
+    Upper,
+    /// ltrim
```

Comment:
Is this `ltrim` or trim? The test seems to indicate `trim`.

Reply:
Yes, sorry, I had actually implemented quite a few more functions (more `ltrim`, `rtrim` and `position`) but then removed them as technically `ltrim` and `rtrim` is not part of the ANSI standard but are Postgres additions (and others).

I think the decision on SQL dialect to support is required before continuing too much further.